lib/credit_card_background.dart

```diff
@@ -33,80 +39,93 @@ class CardBackground extends StatelessWidget {
   final double? height;
   final double padding;
   final BoxBorder? border;
+  final FloatingController? floatingController;
+  final double? glarePosition;
+  final bool shadowEnabled;
 
   @override
   Widget build(BuildContext context) {
     final Orientation orientation = MediaQuery.of(context).orientation;
     return LayoutBuilder(
-        builder: (BuildContext context, BoxConstraints constraints) {
-      final double screenWidth = constraints.maxWidth.isInfinite
-          ? MediaQuery.of(context).size.width
-          : constraints.maxWidth;
-      final double screenHeight = MediaQuery.of(context).size.height;
-      return Stack(
-        alignment: Alignment.center,
-        children: <Widget>[
-          Container(
-            margin: EdgeInsets.all(padding),
-            decoration: BoxDecoration(
-              borderRadius: BorderRadius.circular(8),
-              border: border,
-              gradient: glassmorphismConfig != null
-                  ? glassmorphismConfig!.gradient
-                  : backgroundGradientColor,
-              image: backgroundImage != null && backgroundImage!.isNotEmpty
-                  ? DecorationImage(
-                      image: ExactAssetImage(
-                        backgroundImage!,
-                      ),
-                      fit: BoxFit.fill,
-                    )
-                  : backgroundNetworkImage != null &&
-                          backgroundNetworkImage!.isNotEmpty
-                      ? DecorationImage(
-                          image: NetworkImage(
-                            backgroundNetworkImage!,
+      builder: (BuildContext context, BoxConstraints constraints) {
+        final double screenWidth = constraints.maxWidth.isInfinite
+            ? MediaQuery.of(context).size.width
+            : constraints.maxWidth;
+        final double screenHeight = MediaQuery.of(context).size.height;
+        return Stack(
+          alignment: Alignment.center,
+          children: <Widget>[
+            Container(
+              margin: EdgeInsets.all(padding),
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(8),
+                boxShadow: shadowEnabled && floatingController != null
+                    ? <BoxShadow>[
+                        BoxShadow(
+                          blurRadius: AppConstants.minBlurRadius,
+                          color: AppConstants.defaultShadowColor
+                              .withOpacity(AppConstants.minShadowOpacity),
+                          offset: Offset(
+                            floatingController!.y * 100,
+                            -floatingController!.x * 100 + 8,
```

Comment:
any specific reason to add 8 here? @kavantrivedi  @aditya-chavda

Reply:
Just to have the shadow a below the card widget by 8 pixel. Want me to add comment there expressing the same?